instrumentation/sidekiq/lib/opentelemetry/instrumentation/sidekiq/instrumentation.rb

```diff
@@ -32,7 +32,6 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
         option :trace_launcher_heartbeat,    default: false, validate: :boolean
         option :trace_poller_enqueue,        default: false, validate: :boolean
         option :trace_poller_wait,           default: false, validate: :boolean
-        option :trace_processor_process_one, default: false, validate: :boolean
```

Comment:
Instead of removal, do we have a pattern for deprecating options? Even if we still remove the patch with this PR, maybe this option hangs around to emit a warning at initialization if it has been set?

Reply:
I personally haven't seen any pattern for deprecation in this repo, but if there is I'm more than willing to use it instead of removing this option.